### ISSUE TITLE
Water Quality Investigation Report - Point Stream Field to Nearest Stream Field in Database Once Added

### ISSUE THREAD

**Issue by cstevenson1** [leeensminger/DelDOT-NPDES-Viewer#64]
Per Field Tool Github Bug #309, a new field will be added to the WQI section of the database for field staff to manually enter the name of the applicable stream. Once this is complete, the WQI report should be updated so that Stream in the report is pointed towards the new stream field in the database.

![image](https://user-images.githubusercontent.com/16919898/27434982-8e9f0582-5728-11e7-849c-c2b9d9bb75bf.png)

**Comment by gerrykelly** [comment]
@leeensminger 
not seeing this field

**Comment by cstevenson1** [comment]
Not sure what input is needed from me. As far as I know, Field Tool Github Bug #309 has not been resolved so per the initial comment, I don't think this issue can be addressed until the field is added.

**Comment by leeensminger** [comment]
@gerrykelly this is unfortunately on hold until this field is added to the database.

**Comment by leeensminger** [comment]
@impittman please address after the appropriate schema changes are made

**Comment by impittman** [comment]
@gerrykelly the database edit to adjust the WQI report field has been created. please update the WQI report field in DEV first to pull data from the following field:

deldot_migration2.DBO.WQ_INVESTIGATIONS.NEAREST_STREAM

**Comment by gerrykelly** [comment]
@KCI-Ablowers 
On hold pending #334

**Comment by gerrykelly** [comment]
@KCI-Ablowers   This should be like a 5-minute simple, update but now waiting for info on #302 and #280

**Comment by gerrykelly** [comment]
@gerrykelly 
replace  'pending' as Stream  with [NEAREST_STREAM]

**Comment by gerrykelly** [comment]
@cstevenson1 @KCI-Ablowers 
Undated in dev

**Comment by mjuber-kci** [comment]
Confirmed in dev Viewer. Stream field in WQ Report appears to be populating properly. Push to prod.

![image](https://user-images.githubusercontent.com/16919958/64960298-0cd19580-d861-11e9-9cd5-376c15345529.png)

**Comment by KCI-Ablowers** [comment]
Confirmed by @mjuber-kci. Push to Prod confirmed.

**Comment by gerrykelly** [comment]
@KCI-Ablowers 
Working in prod

**Comment by mjuber-kci** [comment]
Appears to be working in prod viewer. 

![image](https://user-images.githubusercontent.com/16919958/66650514-ce7c8b80-ebfe-11e9-99d0-8585db439722.png)
